routes/admin.js

```diff
@@ -1,9 +1,18 @@
 const express = require('express');
+const hbs = require('hbs');
 
 const router = express.Router();
 
 const adminController = require('../controllers/admin');
 
+hbs.registerHelper({
+  myCategory: (thisCategoryId, myCategories) => {
```

Comment:
myCategory might be misleading... it is checking if the category already belongs to the equipment